src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -0,0 +1,123 @@
+Keywords:
+- sinusoidal
+- execution control
+- insertMany
+- find
+- CrudActor
+
+dbname: &dbname mixed_sinusoidal
+DocumentCount: &NumDocs 5000
+CollectionCount: &NumColls 1
+
+NumPhases: &NumPhases 2
+Document: &doc
+  a: {^RandomInt: {min: 0, max: *NumDocs}}
+  b: {^RandomString: {length: 16}}
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 5100
+
+ActorTemplates:
+- TemplateName: WritesActorWithSleepTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: WritesActorWithSleep}}
+    Type: CrudActor
+    Database: *dbname
+    Threads: {^Parameter: {Name: "Threads", Default: 1000}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          Duration: 3 minutes
+          SleepAfter: 90 seconds
+          Repeat: 3
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: insertOne
+            OperationMetricsName: WriteDocs
+            OperationCommand:
+              Document: *doc
+            ThrowOnFailure: false
+- TemplateName: ReadsActorWithSleepTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: ReadsActorWithSleep}}
+    Type: CrudActor
+    Database: *dbname
+    Threads: {^Parameter: {Name: "Threads", Default: 1000}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: {^Parameter: {Name: "ActivePhases", Default: [1]}}
+        NopInPhasesUpTo: *NumPhases
+        PhaseConfig:
+          SleepBefore: 90 seconds
+          Duration: 3 minutes
+          Repeat: 3
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: find
+            OperationCommand:
+              Filter: { a: { ^RandomInt: { min: 0, max: *NumDocs } } }
+            OperationMetricsName: ReadDocs
+            ThrowOnFailure: false
+Actors:
+- Name: Setup
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        # Have 5k documents ready at start
+        Repeat: 1
+        BatchSize: 100
+        Threads: 1
+        DocumentCount: *NumDocs
+        Database: *dbname
+        CollectionCount: *NumColls
+        Document: *doc
+        Indexes:
+        - keys: {a: 1}
+
+- ActorFromTemplate:
+    TemplateName: WritesActorWithSleepTemplate
+    TemplateParameters:
+      Name: {^Parameter: {Name: "WritingName", Default: WritingActor_50_50}}
+      Threads: {^Parameter: {Name: "WritingThreads", Default: 2500}}
+      ActivePhases: [1]
+
+- ActorFromTemplate:
+    TemplateName: ReadsActorWithSleepTemplate
+    TemplateParameters:
+      Name: {^Parameter: {Name: "ReadingName", Default: ReadsActor_50_50}}
+      Threads: {^Parameter: {Name: "ReadingThreads", Default: 2500}}
+      ActivePhases: [1]
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone
+      - standalone-all-feature-flags
```

Comment:
Will we be running this on more types of configurations later?

Reply:
I confined this to standalones for now since we discussed how flow control might be an issue for creating the right overload circumstances. However, I think we can file a ticket to experiment with more configurations once we get all the workloads running.

Reply:
I feel like this workload would make sense on a 3-node replica set as well (and maybe even 1-node)? Even if it ends up not being super useful for execution control for the time being, that seems like an issue with tuning flow control more than anything else

Reply:
Sure, I've added the corresponding tags to let this run on replica sets.